snapcraft/tests/unit/test_mangling.py

```diff
@@ -106,3 +108,116 @@ def test_following_docstring_no_rewrite(self):
             This is a test
             =======================
         """)))
+
+
+class FakeElf(fixtures.Fixture):
```

Comment:
We have snapcraft/tests/fixture_setup.py-- should this go in there?

Reply:
the only reason I didn't move it into fixtures_setup is because it is so big and I believe it has the wrong name.

I'll propose a followup with a refactor to `fixtures_setup`